README.md

```diff
@@ -32,6 +32,8 @@ maintain the exports in my source file. So someone created a post-build script
 to concatenate them to the end of the file. I built this plugin so I could do
 that without having an ad-hoc post-build script.
 
+Read <a href="https://blog.kentcdodds.com/make-maintainable-workarounds-with-codegen-d34163a09c13">Make maintainable workarounds with codegen<U+1F4A5></a> for more inspiration
```

Comment:
Looks like the emoji didn't get copy/pasted properly.

![screenshot_20171009-090413](https://user-images.githubusercontent.com/1500684/31344800-f2699f5c-acd0-11e7-8abc-d64e687250ae.png)


Also if you could add the `> ` to the start of the line that would be great! Also, instead of an `<a>` tag, could you use markdown? You can just copy/paste the markdown I provided in the issue and that should work 👍